tests/integration/test_cli.py

```diff
@@ -291,3 +291,9 @@ def test_pipenv_outdated_prerelease(PipenvInstance):
             f.write(contents)
         c = p.pipenv('update --pre --outdated')
         assert c.returncode == 0
+
+@pytest.mark.cli
+def test_pipenv_run_with_special_chars(PipenvInstance):
+    with PipenvInstance():
+        c = subprocess_run(["pipenv", "run", "echo", "[3-1]"])
```

Comment:
You should test this in the windows module: `tests/integration/test_windows.py` on Linux this test always passes.

Reply:
OK, I fixed it.

Reply:
Thank you very much!